### PR TITLE
Contact page

### PR DESCRIPTION
**Pull request by KlaraGajaszek**
closing #49

### CODE REVIEW

File: src/pages/contact/index.tsx [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+    (event) => {
+      const { value } = event.target;
+      setFieldValue(name, value);
+    },
+    [setFieldValue, name],
+  );
+  return <TextField {...fieldToTextField(props)} {...textFieldData} onChange={onChange} />;
+}
+
+interface FormValues {
```
Comment (KonradSzwarc) [inline src/pages/contact/index.tsx]: Use type instead

File: src/pages/contact/index.tsx [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+
+const initialValues: FormValues = {
+  fullName: '',
+  email: '',
+  title: '',
+  messageContent: '',
+};
+
+const validationSchema = Yup.object().shape({
+  fullName: Yup.string().required('Full Name is required field'),
```
Comment (KonradSzwarc) [inline src/pages/contact/index.tsx]: Use `.label()` to provide the field's name so it wouldn't be necessary to add a custom message

File: src/pages/contact/index.tsx [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+  const developerProfile = useDeveloperProfile();
+  const { componentType, isDesktop, isMobile, isTablet } = useComponentType();
+
+  return (
+    <Container className={classes.container} maxWidth="lg">
+      <Helmet>
+        <title>Contact page</title>
+      </Helmet>
+      {isDesktop && (
+        <Box className={classes.detailsCard}>
```
Comment (KonradSzwarc) [inline src/pages/contact/index.tsx:204]: This card should be defined as a separate component with all its props

File: src/pages/contact/index.tsx [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+            isFreelancer={developerProfile.isFreelancer}
+            resumeLink={developerProfile.cv}
+            type={componentType}
+          />
+        </Box>
+      )}
+
+      <Box className={classes.main}>
+        {isDesktop && (
+          <Navbar
```
Comment (KonradSzwarc) [inline src/pages/contact/index.tsx:227]: Navbar should be defined as a separate component with all its props

File: src/pages/contact/index.tsx [hunk 5]
```diff
@@ -0,0 +1,298 @@
...
+import { Navbar } from '../../components/Navbar';
+import { SectionTitle } from '../../components/SectionTitle';
+import { TextField } from '../../components/TextField';
+import { useDeveloperProfile } from '../../containers/DeveloperProfile';
+import { useComponentType } from '../../hooks/useComponentType';
+import { textFieldData } from '../../views/contact-page/data';
+
+const contactPageItemShadow = '0 40px 50px 0 rgba(103, 118, 128, 0.1)';
+
+function TextFieldWrap(props: TextFieldProps) {
```
Comment (KonradSzwarc) [inline src/pages/contact/index.tsx]: Why do we need to define our custom wrapper for the TextField? Couldn't we just use one from the formik-material-ui library?

**Review by KonradSzwarc — APPROVED** [review]

File: src/pages/contact/index.tsx (hunk 3 above)
Comment (KlaraGajaszek) [reply src/pages/contact/index.tsx:204]: I created new issues

File: src/pages/contact/index.tsx (hunk 4 above)
Comment (KlaraGajaszek) [reply src/pages/contact/index.tsx:227]: I created new issues

**Pull request merged**